apps/notifications/src/panel/templates/summary-in-single.jsx

```diff
@@ -61,11 +68,55 @@ class UserHeader extends Component {
 	}
 }
 
+class WritingPromptHeader extends Component {
+	render() {
+		const icon = this.props.site.media[ 0 ];
+		const img_tag = (
+			<img src={ icon.url } height={ icon.height } width={ icon.width } alt="prompt-icon" />
+		);
+		const home_url = this.props.site.ranges[ 0 ].url;
+		const settings_url =
```

Comment:
hmmm is there not a more elegant way to do this?

Reply:
There might be but I borrowed this from existing pattern. We can look at this again if its an issue.

Reply:
Wasn't sure which line you meant! I've replaced the string replace with `withoutHttp` as it appears neater.